### PR TITLE
Scope exports from Grapher util

### PR DESCRIPTION
**Pull request by handeyeco**
## Summary:
Several things in `grapher/util.ts` were exported out of `perseus`, but were only used within `perseus`.

## Test plan:
Nothing should change

### CODE REVIEW

**Review by jeremywiebe — COMMENTED** [review]
I'm not sure this change is an improvement.  I don't disagree that it cleans up exports, but it comes at the cost of a muddied set of base exports from `perseus`. For instance, if I import `chooseType` from `@khanacademy/perseus` it's very unclear what that does and that it's a helper function for Grapher.

File: packages/perseus-editor/src/widgets/grapher-editor.tsx [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
     Changeable,
-    GrapherUtil,
     GrapherWidget,
     SizingUtils,
+    DEFAULT_GRAPHER_PROPS,
+    chooseType,
+    defaultPlotProps,
+    getEquationString,
+    allTypes,
+    typeToButton,
```
[the remark this answers is not in the document]
Comment (handeyeco) [reply packages/perseus-editor/src/widgets/grapher-editor.tsx:12]: We could do:

```
// index.ts
import {
    allTypes,
    DEFAULT_GRAPHER_PROPS,
    chooseType,
    defaultPlotProps,
    getEquationString,
    typeToButton,
} from "./widgets/grapher/util";

export const GrapherUtil = {
    allTypes,
    DEFAULT_GRAPHER_PROPS,
    chooseType,
    defaultPlotProps,
    getEquationString,
    typeToButton,
};
```

I think this solves the same problem.

[the remark this answers is not in the document]
Comment (jeremywiebe) [reply packages/perseus-editor/src/widgets/grapher-editor.tsx:12]: Yep.  That would work.  Within Perseus we'd just import by the named export. Outside, we could import `GrapherUtil` and deconstruct from there.  I suspect this is all exported for the editors (another reason to think further about merging).

File: packages/perseus/src/index.ts [hunk 2]
```diff
@@ -36,7 +36,23 @@ export {default as TableWidget} from "./widgets/table";
...
+import {
+    allTypes,
+    DEFAULT_GRAPHER_PROPS,
+    chooseType,
+    defaultPlotProps,
+    getEquationString,
+    typeToButton,
+} from "./widgets/grapher/util";
+
+export const GrapherUtil = {
```
Comment (jeremywiebe) [inline packages/perseus/src/index.ts:48]: Love it.  Could you just add a comment on this export explaining why we're doing this (namely, the exported functions make more sense as they're scoped inside the `GrapherUtil` object).

**Review by jeremywiebe — APPROVED** [review]

**Pull request merged**